webviz_subsurface/plugins/_volumetric_analysis/views/fipfile_qc_layout.py

```diff
@@ -0,0 +1,83 @@
+from typing import List
+import pandas as pd
+
+import dash_html_components as html
+import webviz_core_components as wcc
```

Comment:
```suggestion

import pandas as pd

from dash import html
import webviz_core_components as wcc
```

Reply:
Should be fixed now 👍 but I put pandas just above dash.. that is correct right?

Reply:
Looks good :) The convention is std.lib before other packages before itnernal in this package :)

https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.